ldes-fragmentisers/ldes-fragmentisers-common/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/FragmentationStrategyImplTest.java

```diff
@@ -22,13 +29,17 @@ class FragmentationStrategyImplTest {
 
 	@Test
 	void when_memberIsAddedToFragment_FragmentationStrategyImplSavesUpdatedFragment() {
-		Fragment fragment = new Fragment(new LdesFragmentIdentifier(new ViewName("collectionName", "view"),
-				List.of()));
+		Fragment fragment = new Fragment(FRAGMENT_ID);
 		Member member = mock(Member.class);
-		when(member.id()).thenReturn("memberId");
+		when(member.id()).thenReturn(MEMBER_ID);
+		when(member.sequenceNr()).thenReturn(SEQ_NR);
 
-		fragmentationStrategy.addMemberToFragment(fragment, member.id(), member.model(), any());
+		List<BucketisedMember> members = fragmentationStrategy.addMemberToFragment(fragment, member, mock(Observation.class));
 
-		verify(fragmentRepository, times(1)).incrementNrOfMembersAdded(fragment.getFragmentId());
+		assertThat(members).hasSize(1);
+		assertThat(members.getFirst()).hasFieldOrPropertyWithValue("memberId", MEMBER_ID)
+				.hasFieldOrPropertyWithValue("viewName", VIEW_NAME)
+				.hasFieldOrPropertyWithValue("fragmentId", FRAGMENT_ID.asDecodedFragmentId())
+				.hasFieldOrPropertyWithValue("sequenceNr", SEQ_NR);
```

Comment:
A little suggestion: an expectedBucketisedMember could be declared with all these properties and then the assertion could look like this:
```java
        assertThat(members.getFirst())
                .usingRecursiveComparison() // this makes sure all fields are compared
                .isEqualTo(expectedBucketisedMember);
```